### ISSUE TITLE
Something wrong with Generation?

### ISSUE THREAD

**Issue by ghost** [ebolam/KoboldAI#312]
Hi there, I mostly leave things untouched except changing temp to be lower but looks generation seems off and also this happens.   I'm using pythia 13B
![Screenshot 2022-11-26 112545](https://user-images.githubusercontent.com/113113106/204070821-2b192b75-da92-40bc-8117-de79a0823bf8.png)

![Screenshot 2022-11-26 112607](https://user-images.githubusercontent.com/113113106/204070759-11cb7803-183d-4589-9d51-a19faca121bf.png)

**Comment by ghost** [comment]
Closed it. Looks like rep pen was set to 1.8 something, don't know why as I set it at 1.1 before.

**Comment by ghost** [comment]
Reopened it again, seems like it's spouting unrelated text despite lowering temp at 0.5 and normal rep pen value (1.1). For context, this is the first time I have this happened with the model and such a low temp.
![Screenshot 2022-11-26 1125245](https://user-images.githubusercontent.com/113113106/204070993-f7aa30bd-d7c8-4d12-89c1-923af7e151a1.png)

**Comment by ghost** [comment]
Another one, not really sure if its the model or what, but haven't really have much problems on united with the same settings.
![aas](https://user-images.githubusercontent.com/113113106/204120888-54e3de24-ddfc-4925-8452-7938f4abcf72.png)

**Comment by ebolam** [comment]
Does it do this in United with the same story/settings?

**Comment by ghost** [comment]
@ebolam  No, I tried it again in united, I can confirm it only happens in the UI2. Literally just after prompt it spouts something like this. I believe some are having the same issues in the discord channels, blaming pythia for this but they used UI2, as for myself, haven't really had problems with it in united until I tried it in UI2.

![Screenshot 2022-11-28 030714](https://user-images.githubusercontent.com/113113106/204154956-ae5d02dd-08a7-4bb2-8699-3eac7179b209.png)

![Screenshot 2022-11-28 030651](https://user-images.githubusercontent.com/113113106/204154847-e9796ec2-9289-40a8-998b-db4c1085512a.png)

**Comment by ebolam** [comment]
Were your settings the same?

**Comment by ghost** [comment]
For what I tried in united and the last image, yes. Haven't got the chance
to upload the one in united since it's fine for my liking.

**Comment by ghost** [comment]
![Screenshot 2022-12-23 122329](https://user-images.githubusercontent.com/113113106/209270553-ad679fcd-141f-4b2e-afb3-0aa440014bc3.png)
Also something like this happens on Pythia 13b Deduped, I never had issues like this on the original UI with the same settings.

**Comment by ghost** [comment]
![Screenshot 2022-12-23 192102](https://user-images.githubusercontent.com/113113106/209327799-e2aa573a-5504-41a0-a511-86413af17e3a.png)
Also notice the space here and in the previous image, I didn't add those and I believe it doesn't happen on UI1 on pythia deduped.

**Comment by ghost** [comment]
![Screenshot 2022-12-23 195316](https://user-images.githubusercontent.com/113113106/209335571-43c5da4a-b695-47d7-a2f2-470aba0c8334.png)
Also double spaces.

**Comment by ghost** [comment]
[Setting.txt](https://github.com/ebolam/KoboldAI/files/10295199/Setting.txt)
Here is my output in UI1, it's fine, not having those problems.
